samza-azure/src/main/java/org/apache/samza/ProcessorEntity.java

```diff
@@ -38,11 +39,13 @@ public ProcessorEntity(String jobModelVersion, String processorId) {
     this.partitionKey = jobModelVersion;
     this.rowKey = processorId;
     this.isLeader = false;
+    Random rand = new Random();
+    this.liveness = rand.nextInt(10000) + 2;
```

Comment:
The liveness is a random value that is written to an attribute of the row in order to update its last modified since timestamp. The random value is there just to update the row in some sense, to assert to the leader that the processor is alive.